learning_assistant/platform_imports.py

```diff
@@ -0,0 +1,47 @@
+"""
+Contain all imported functions coming out of the platform.
+
+We know these functions will be available at run time, but they
+cannot be imported normally.
+"""
+
+
+def get_text_transcript(video_block):
+    """Get the transcript for a video block in text format, or None."""
+    # pylint: disable=import-error, import-outside-toplevel
+    from xmodule.exceptions import NotFoundError
+    from xmodule.video_block.transcripts_utils import get_transcript
+    try:
+        transcript, _, _ = get_transcript(video_block, output_format='txt')
+    except NotFoundError:
+        # some old videos have no transcripts, just accept that reality
+        return None
+    return transcript
+
+
+def get_single_block(request, user_id, course_id, usage_key_string, course=None, will_recheck_access=False):
```

Comment:
I don't think we'll ever want this to be `True` because then we won't be checking learner access against to the block. I think it would be better not to expose this function parameter in that case, since we don't want someone to accidentally pass `True`. What do you think?

Reply:
Oh, that's a great point! I'll update this function.